docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -190,6 +190,22 @@ When diagnosing the health state of the task, you will most likely be interested
 
 Investigating the underlying task can help you gauge whether the problem you’re seeing is rooted in the rule not running at all, whether it’s running and failing, or whether it is running, but exhibiting behavior that is different than what was expected (at which point you should focus on the rule itself, rather than the task).
 
+[discrete]
+[[alerting-kibana-disable]]
+=== Temporarily disable
+
+A cluster may become unresponsive or sluggish if too many or expensive {kib} rules 
```

Comment:
It is not obvious what an "expensive" rule is.  I recommend removing that term or else adding information about what makes a particular rule expensive:
```suggestion
A cluster may become unresponsive or sluggish if too many {kib} rules 
```